package.json

```diff
@@ -40,6 +40,7 @@
     "react": "^0.14 || ^15"
```

Comment:
oh, almost forgot. Pure component requires react@15.3, so `peerDependencies` should be updated to have `"react": ">=15.3"`.